pom.xml

```diff
@@ -188,7 +188,7 @@
             <!-- use the scm publish plugin to deploy to GitHub -->
             <plugin>
                 <artifactId>maven-scm-publish-plugin</artifactId>
-                <version>3.3.0</version>
+                <version>3.3.1-SNAPSHOT</version>
```

Comment:
This should be reverted to 3.3.0.

Reply:
On the other hand - there is no pages (even no `gh-pages` branch) for this repository, so unsure if there is anything to be published. Perhaps this plugin should be dropped.

Reply:
Oh, you are right, I see! :D
And I will remove that part from the script.